compiler/rustc_codegen_llvm/messages.ftl

```diff
@@ -76,8 +76,8 @@ codegen_llvm_target_machine = could not create LLVM TargetMachine for triple: {$
 codegen_llvm_target_machine_with_llvm_err = could not create LLVM TargetMachine for triple: {$triple}: {$llvm_err}
 
 codegen_llvm_unknown_ctarget_feature =
-    unknown feature specified for `-Ctarget-feature`: `{$feature}`
-    .note = it is still passed through to the codegen backend
+    unknown and unstable feature specified for `-Ctarget-feature`: `{$feature}`
```

Comment:
"and"? or "or"?

Reply:
"and"

Reply:
Why? I don't totally understand how something can both be unknown and unstable.

Reply:
If it's unknown it's clearly not stable, and hence it is unstable.

Reply:
Why does that need clarification? Like, if it's clearly not stable by virtue of being uknown, why are we saying it's unknown *and* unstable?

Reply:
I think it's worth emphasizing that unknown implies unstable. Unstable has a particular meaning in the Rust context, and that implication is very clear to us, but might not be clear at all to users.

It is extremely unusual that we allow unstable things to even be done on stable Rust, making it even more important to point this out explicitly.